index.html

```diff
@@ -31,5 +31,12 @@ <h4 class="text feature-title"> No internet needed. </h4>
         </div>
       </div>
     </div>
+    <div class="shadow">
```

Comment:
there's no  need to have a div for this, you can apply the shadow property to the footer directly